Add osu! Philippines Nationals 2022-2023 wiki article (#10331)
ppy/osu-wiki: wiki/Tournaments/oPHN/2023_osu/en.md

- ::{ flag=PH }:: [okirema](https://osu.ppy.sh/users/12544805)
- ::{ flag=PH }:: [Buizie](https://osu.ppy.sh/users/4795670)
- ::{ flag=PH }:: [Flakens](https://osu.ppy.sh/users/13583261)
- ::{ flag=PH }:: [\[ x \]](https://osu.ppy.sh/users/14942720)
- ::{ flag=PH }:: [LeFerraille](https://osu.ppy.sh/users/11001039)
- ::{ flag=PH }:: [xenonical1st](https://osu.ppy.sh/users/12278219)
- ::{ flag=PH }:: [adrius](https://osu.ppy.sh/users/12991835)
- ::{ flag=PH }:: [T3nomi](https://osu.ppy.sh/users/18374145)
- ::{ flag=PH }:: [zSidd](https://osu.ppy.sh/users/14146602)
- ::{ flag=PH }:: [Lanskyy](https://osu.ppy.sh/users/20660835)

## Podium

This competition has come to an end and resulted in the following podium:

| Placing | Player |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=PH }:: [NathanRam1918](https://osu.ppy.sh/users/4734703) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=PH }:: [Xyloz](https://osu.ppy.sh/users/12040280) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=SA }:: [-Peachii](https://osu.ppy.sh/users/9683186) |

## Mappools

> ### Grand finals

Reviewer: ```suggestion
### Grand Finals
```

  1. [PassCode - It's you (nemidnight) [Extreme]](https://osu.ppy.sh/beatmapsets/1633506#osu/3334163)
  2. [Xi - Youyoumu ~ Run or Dash (GYGY) [Extra]](https://osu.ppy.sh/beatmapsets/1782703#osu/3650832)
  3. [Official HIGE DANdism - Mixed Nuts (Amateurre) [Liyuuchie's Ascent Extra]](https://osu.ppy.sh/beatmapsets/1802422#osu/3695972)
  4. [GHOSTEMANE - Mercury: Retrograde (Anishina Kodoku) [Thelema]](https://osu.ppy.sh/beatmapsets/838193#osu/1754868)
- Hidden
  1. [UKRampage - Jack-the-Ripper (DeviousPanda) [Callie's Expert]](https://osu.ppy.sh/beatmapsets/1333932#osu/2851425)
  2. [Osamu Kubota - Presto (Kloyd) [ponbot's Extra]](https://osu.ppy.sh/beatmapsets/1452332#osu/3008005)
- Hard Rock
  1. [NIWASHI - Playing with Ruby (Down) [Extra]](https://osu.ppy.sh/beatmapsets/1306570#osu/2823535)
  2. [Spide - Ama no Hane wa Kami no Gotoshi (tatemae) [Intertextuality]](https://osu.ppy.sh/beatmapsets/926835#osu/1935844)
- Double Time
  1. [Suzumiya Haruhi (CV: Hirano Aya) - God knows... (Luscent) [Insane]](https://osu.ppy.sh/beatmapsets/1650000#osu/3367803)
  2. [KARUT - Glitch Street (Shurelia) [Kowari's Insane]](https://osu.ppy.sh/beatmapsets/1817644#osu/3783845)

## Match results

> ### Grand finals

Reviewer: ```suggestion
### Grand Finals
```

| **-Peachii** ::{ flag=PH }:: | **5** | 1 | ::{ flag=PH }:: Hinatsuru Ai | [#1](https://osu.ppy.sh/community/matches/106245079) |
| scuri ::{ flag=PH }:: | 0 | **5** | ::{ flag=PH }:: **bluezenithgames** | *win by default* |
| **Impowster** ::{ flag=PH }:: | **5** | 1 | ::{ flag=PH }:: Splenden | [#1](https://osu.ppy.sh/community/matches/106270740) |
| **LordGabriel** ::{ flag=PH }:: | **5** | 1 | ::{ flag=PH }:: Binninja | [#1](https://osu.ppy.sh/community/matches/106266788) |
| **Yukixo** ::{ flag=PH }:: | **5** | 1 | ::{ flag=PH }:: OsuSeulgiDragon | [#1](https://osu.ppy.sh/community/matches/106246349) |
| **Xyloz** ::{ flag=PH }:: | **5** | 3 | ::{ flag=PH }:: LeFerraille | [#1](https://osu.ppy.sh/community/matches/106269426) |
| **-Myhra** ::{ flag=PH }:: | **5** | 4 | ::{ flag=PH }:: Xyphox | [#1](https://osu.ppy.sh/community/matches/106263488) |
| **rwnd_** ::{ flag=PH }:: | **5** | 2 | ::{ flag=PH }:: CircIes | [#1](https://osu.ppy.sh/community/matches/106245042) |
| **NathanRam1918** ::{ flag=PH }:: | **5** | 0 | ::{ flag=PH }:: caetrn | [#1](https://osu.ppy.sh/community/matches/106285090) |

## Rulesets

### Registration

> - Only players with the Philippine flag on their profile are eligible to play. Players who don’t have the Philippine flag on their profile can present a proof of their Philippine Citizenship to the staff in order to be eligible to play.

Reviewer 2: ```suggestion
- Only players with the Philippine flag on their profile are eligible to play. Players who don't have the Philippine flag on their profile can present a proof of their Philippine Citizenship to the staff in order to be eligible to play.
```

> - Staff Members **who are NOT Referees, Mappool Selectors, Hosts, nor are part of any managing role** are permitted to join the tournament. [Read the FAQ](/wiki/Tournaments/Official_support#faq) about what is considered as “staff” for more details.

Reviewer 2: ```suggestion
- Staff Members **who are NOT Referees, Mappool Selectors, Hosts, nor are part of any managing role** are permitted to join the tournament. [Read the FAQ](/wiki/Tournaments/Official_support#faq) about what is considered as "staff" for more details.
```

- Players must be at least rank **#50,000** and up to join.
- Registrations are done by completing this [Google form](https://docs.google.com/forms/d/13t0fmVrBkAq6bZgOQcuCjVx7LbhB_BqSxZ7tLgZbtG8/).
- Players are required to join the [Discord server](https://discord.gg/jn7E2gCnpb).
- All players will undergo screening after the closing of registrations.

### Tournament format

> The tournament format was 1v1, double-elimination tournament. Players play through qualifiers, and the top 32 players will proceed to the Round of 32. 

Reviewer 2: ```suggestion
The tournament format was 1v1, double elimination. Players play through Qualifiers, and the top 32 players will proceed to the Round of 32. 
```

> Players get 1 ban on RO32 and RO16 and 2 bans on quarterfinals onwards. All matches are played with Score v2 and NF, head-to-head mode.

Reviewer 2: ```suggestion
Players get 1 ban on RO32 and RO16 and 2 bans on Quarterfinals onwards. All matches are played with ScoreV2 and NF in Head-to-head mode.
```

### Pool size

> Note: The star ratings listed below are the range of  star ratings per round.

Reviewer 2: ```suggestion
Note: The star ratings listed below are the range of star ratings per round.
```

> | Round stage | Best of | NM | HD | HR | DT | TB | Star rating |

Reviewer 2: ```suggestion
| Round | Best of | NM | HD | HR | DT | TB | Star rating |
```

| :-- | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
| Qualifiers | - | 4 | 2 | 2 | 2 | - | 5.0*-6.6* |
| Round of 32 | 9 | 5 | 3 | 3 | 3 | 1 | 4.8*-5.95* |
| Round of 16 | 9 | 5 | 3 | 3 | 3 | 1 | 5.0*-6.3* |
| Quarterfinals | 11 | 5 | 4 | 4 | 4 | 1 | 5.2*-6.6* |
| Semifinals | 11 | 5 | 4 | 4 | 4 | 1 | 5.5*-6.8* |
| Finals | 13 | 6 | 4 | 4 | 4 | 1 | 5.8*-7.1* |

> | Grand finals | 13 | 6 | 4 | 4 | 4 | 1 | 5.9*-7.3* |

Reviewer 2: ```suggestion
| Grand Finals | 13 | 6 | 4 | 4 | 4 | 1 | 5.9*-7.3* |
```

> ### Tournament Rules

Reviewer 2: ```suggestion
### Tournament rules
```

> - All schedules in the tournament are default to Philippine Standard Time (+8 UTC).

Reviewer 2: ```suggestion
- All schedules in the tournament are default to Philippine Standard Time (UTC+8).
```

- Reschedules can only be done anytime before 24 hours of the match unless there is an emergency.
- A player will be given ten (10) minutes of grace period after the match time to enter the lobby. Failure to join after the grace period ends will result in a forfeit and the opposing player will be awarded with a **win by default**. 
- If both players did not show up after the grace period, the higher seed will be awarded with a **win by default** and will proceed to the next round.
- Banning two maps from the same mod category is not allowed.
- Any player who displays unsportsmanlike behavior will receive a warning/disqualification.

> ### Qualifier Rules

Reviewer 2: ```suggestion
### Qualifier rules
```

- Qualifiers pool will be released after the player screening.
- Players will have to sign up to a lobby. Ensure you have enough time to play through the pool.
- Maps are played **once** in order (NM, HD, HR, DT).
- A reschedule is forced when the player does not make his lobby on time.
- In case of disconnection (while the map was being played), the player may replay the missed map **one** time later on.
- The top 32 players by **average map rank** will proceed to RO32.

> ### Match Procedure

Reviewer 2: ```suggestion
### Match procedure
```

- Referees will notify and invite both players into the lobby before the designated schedule. 
- There will be no warmups.

> - Players will “!roll” and the winner decides between first pick, second ban or second pick, first ban.

Reviewer 2: ```suggestion
- Players will `!roll` and the winner decides between first pick, second ban or second pick, first ban.
```
